client/a8c-for-agencies/sections/purchases/licenses/license-details/actions.tsx

```diff
@@ -59,6 +62,8 @@ export default function LicenseDetailsActions( {
 	const isPressableLicense = isPressableHostingProduct( licenseKey );
 	const isWPCOMHostingLicense = isWPCOMHostingProduct( licenseKey );
 	const pressableManageUrl = 'https://my.pressable.com/agency/auth';
+	const isCancellationScheduled =
```

Comment:
For clarity in the code, I think that if we're referring to auto-renew being disabled/enabled then we should call it something that reflects that.